_data/projects/piment-css.yml

```
@@ -0,0 +1,12 @@
+name: Piment Css
+desc: Piment is a css micro-framework made by Webmonster Antilles.
+site: https://pimentcss.webmonster.tech/
+tags:
+  - css
+  - framework
+  - starter
+  - kit
+  - gulp
+upforgrabs:
+  name: Piment-Css
+  link: https://github.com/WebmonsterA/Piment-Css
```

:art this section should point to a label and URL to a list of issues - does this seem like a suitable label to use?

```suggestion
  name: good first issue
  link: https://github.com/WebmonsterA/Piment-Css/labels/good%20first%20issue
```

Also, there are no issues active on the project, which means the project will become unlisted. Do you have plans to add tasks for new contributors?

---

Sorry i dont understand, i have no issue at this time